plugins/source/azure/client/services/web.go

```diff
@@ -9,20 +9,25 @@ import (
 )
 
 type WebClient struct {
-	Apps AppsClient
+	Apps             WebAppsClient
+	SiteAuthSettings WebSiteAuthSettingsClient
 }
 
-type AppsClient interface {
+type WebAppsClient interface {
 	List(ctx context.Context) (result web.AppCollectionPage, err error)
 	ListPublishingProfileXMLWithSecrets(ctx context.Context, resourceGroupName string, name string, publishingProfileOptions web.CsmPublishingProfileOptions) (result web.ReadCloser, err error)
-	GetAuthSettings(ctx context.Context, resourceGroupName string, name string) (result web.SiteAuthSettings, err error)
 	GetVnetConnection(ctx context.Context, resourceGroupName string, name string, vnetName string) (result web.VnetInfo, err error)
 }
 
+type WebSiteAuthSettingsClient interface {
+	GetAuthSettings(ctx context.Context, resourceGroupName string, name string) (result web.SiteAuthSettings, err error)
+}
+
 func NewWebClient(subscriptionId string, auth autorest.Authorizer) WebClient {
 	apps := web.NewAppsClient(subscriptionId)
 	apps.Authorizer = auth
 	return WebClient{
-		Apps: apps,
+		Apps:             apps,
+		SiteAuthSettings: apps,
```

Comment:
Again I created a new `WebSiteAuthSettingsClient` to keep things consistent instead of having auth settings use the apps client (see [here](https://github.com/cloudquery/cloudquery/blob/973af8b82b59e0c3af7ae031c866888fea1edbca/plugins/source/azure/resources/services/web/app_auth_settings.go#L321))